ci/autoscaler/tasks/update-sdk/task.yml

```diff
@@ -4,7 +4,7 @@ platform: linux
 image_resource:
   type: registry-image
   source:
-    repository: ghcr.io/cloudfoundry/app-autoscaler-release-deploy
+    repository: ghcr.io/cloudfoundryapp-autoscaler-release-tools
```

Comment:
Typo: missing / in the path

Reply:
Nicely spotted, thanks!

Reply:
Fixed in https://github.com/cloudfoundry/app-autoscaler-release/pull/2756/commits/ea19c0f903413c52d3fc7c66468ab5bc13fb3fc5